sklearn/tests/test_naive_bayes.py

```diff
@@ -114,6 +114,16 @@ def test_gnb_priors():
     assert_array_almost_equal(clf.class_prior_, np.array([0.3, 0.7]))
 
 
+def test_gnb_priors_sum_isclose():
+    """Test whether the class prior sum is properly tested"""
```

Comment:
Use a comment rather than a docstring here. We do this because nose hides the function name when docstring is used.